tracer/src/Datadog.Trace.Tools.Runner/Checks/ProcessBasicCheck.cs

```diff
@@ -147,7 +178,33 @@ internal static bool CheckRegistry(IRegistryService? registry = null)
                     }
                 }
 
-                return !foundKey;
+                if (foundKey)
+                {
+                    return false;
+                }
```

Comment:
Why are we short-circuiting here, and doing the subsequent checks? 🤔 The `foundKey` is determining whether we found a "suspicious" key, but should we not do the other checks too, even if we found them? "suspicious" doesn't necessarily mean "wrong" right, and might imply there's no other issues?

Reply:
Good point, there is no reason not to run the other registry checks

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/2482/commits/6de07d2cfa2e2fb34a576ec9fce8c411dace42c7